lib/puppet/provider/package/gem.rb

```diff
@@ -90,8 +90,14 @@ def insync?(is)
     is.any? { |version| dependency.match?('', version) }
   end
 
+  def gem_version(cmd)
+    command = [cmd, '--version']
+    execute(command, {:failonfail => true, :combine => true, :custom_environment => {"HOME"=>ENV["HOME"]}})
```

Comment:
I didn't follow this part, why pass `failonfail`, etc?

Reply:
Sorry, that comes from newer branches, like:

https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/package/gem.rb#L134

I will revert to use `execute(command)` to match this branch.